frontend/components/httpsUrlInput.js

```diff
@@ -13,7 +13,7 @@ const HttpsUrlInput = ({ placeholder, ...props }) => (
   <input
     {...props}
     type="url"
-    pattern="https://.+"
+    pattern="https://.+\.\w{2,}"
```

Comment:
Do we need to add something to match path components (in case someone wants to use a path, like handbook is doing)?